src/main/help_menu.ts

```diff
@@ -0,0 +1,33 @@
+export function getHelpMenu() {
+
+    console.log(`
```

Comment:
```suggestion
/* ignore-console-log */
    console.log(`
```
To make the tests pass which normally only allows using our own logger, except when using this special comment

Reply:
ok, I'll change it, run the commands, and add an entry to the changeling.md thanks!